target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -545,6 +545,15 @@ yuncore,a770)
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "green:lan" "switch0" "0x10"
 	;;
+yuncore,cpe830)
+	ucidef_set_led_netdev "wan" "WAN" "wan" "eth1"
+	ucidef_set_led_netdev "lan" "LAN" "lan" "eth0"
+	ucidef_set_rssimon "phy0-sta0" "200000" "1"
+	ucidef_set_led_rssi "wifi_low" "WIFILOW" "green:signal0" "phy0-sta0" "1" "100"
```

Comment:
Use `wlan0` instead of `phy0-sta0`, just like the other devices. The default wlan mode is AP on openwrt.

Reply:
Done